### PR TITLE
iframe embed code in Core CMS templates

### PR DESCRIPTION
**Pull request by taoteg**
## Overview: ##

Trying to better integrate the iFrame into the CMS pages.

## Related Jira tickets: ##

* [COOKS-350](https://jira.tacc.utexas.edu/browse/COOKS-350)

## Summary of Changes: ##

Only one minor edit to code that is of no impact.
The changes are in the iframe snippet code as I try and ID the best setup.
The base template (full-width) should be adequate for our goals.

## Testing Steps: ##

1. Pull the branch, do a clean build and run the project as usual.
2. When populating the iframe snippets in the CMS admin, use this code instead:

```
<div style="margin:0; padding:0;">
    <div style="display:flex; flex-direction:column; min-height:100vh;">
        <iframe style="border:none; flex-grow:1;" src="https://cep.test/protx/dash/analytics"></iframe>
    </div>
</div>
```

## Notes: ##

- This change adds back the ability of the iframe page to grow/scroll vertically while constraining it horizontally to the page width.
- It uses less snippet code and is simpler.
- Flex-box is the recommended approach for dynamic embedded content sizing.

## Issues: ##

- None

## UI Photos:

- None

### CODE REVIEW

**Comment by nathanfranklin** [comment]
LGTM. 💯  Can the updated snippet be added to the README? 

(he <> in this PR was in another refactor.)

**Comment by taoteg** [comment]
Update - Ignore the testing steps above, they were pre-single-view integration.
Just run as usual, add the code from the README, and test.

File: README.md [hunk 1]
```diff
@@ -48,30 +48,48 @@ Get access to Django CMS, run `docker exec -it core_portal_cms /bin/bash` and th
...
+* Add a `New Snippet`.
+* Name it `Data and Analysis`.
+* Copy & paste the following markdown code into the body field:
 ```
-<body style="margin:0px;padding:0px;overflow:hidden">
-    <div style="position:relative;width:100vw;height:100vh;">
-        <iframe src="https://cep.test/protx/dash/" frameborder="0" allowfullscreen style="position:absolute;top:0;left:0;width:100%;height:100%;" width="100%" height="100%"></iframe>
+<div style="margin:0; padding:0;">
+    <div style="display:flex; flex-direction:column; min-height:100vh;">
+        <iframe style="border:none; flex-grow:1;" src="https://cep.test/protx/dash/analytics"></iframe>
```
Comment (nathanfranklin) [inline README.md:62]:
```suggestion
        <iframe style="border:none; flex-grow:1;" src="https://cep.test/protx/dash/"></iframe>
```
Recent changes make this path to be just dash/

**Review by nathanfranklin — APPROVED** [review]
LGTM. 

**Just one tweak on the iframe src** is needed before merging.

I really dig the clear instructions of setting it up in CMS as that trips me up and will be GREAT for any devs onboarding.

File: README.md (hunk 1 above)
Comment (taoteg) [reply README.md:62]: Werd!

**Pull request merged**